### PR TITLE
update 1.15.4

### PR DESCRIPTION
**Pull request by Arishamays1**
## ☆ Cornac 1.15.4 Update ☆
[Jira Ticket](https://anaconda.atlassian.net/browse/PKG-2473)
[Upstream](https://github.com/PreferredAI/cornac)
[Dependencies ](https://github.com/PreferredAI/cornac/blob/master/setup.py)
# Changes
- Updated version number and `sha256`
- Updated pinnings on dependencies
- Added `libgomp` library for `linux` support
- Updated `About` section

### CODE REVIEW

**Review by varlackc — COMMENTED** [review]
In order to allow users of different languages to read the documentation in their local language we probably should change the `doc_url`

  from
  https://cornac.readthedocs.io/en/latest/
  
  into the following:

  doc_url: https://cornac.readthedocs.io

  While the change will require a redirect, it should be ok, because users of other languages will be able to read the documentation in their preferred language.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,53 +1,64 @@
...
 
 build:
-  skip: true  # [py<35]
-  number: {{ build }}
-  script: {{ PYTHON }} -m pip install . -vv
-
+  skip: True  # [py<38]
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+  # missing_dso_whitelist:     # [win64]
```
Comment (M-Waszkiewicz-Anaconda) [inline recipe/meta.yaml]: remove

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,53 +1,64 @@
...
+  #   - '_openmp_mutex'     # [win64]
```
Comment (M-Waszkiewicz-Anaconda) [inline recipe/meta.yaml]: remove

**Review by M-Waszkiewicz-Anaconda — APPROVED** [review]
remove some unneeded comments, otherwise LGTM

File: recipe/meta.yaml (hunk 2 above)
Comment (lorepirri) [reply recipe/meta.yaml]:
```suggestion
```

File: recipe/meta.yaml (hunk 1 above)
Comment (lorepirri) [reply recipe/meta.yaml]:
```suggestion
```

**Review by lorepirri — APPROVED** [review]
pinnings in `host`, otherwise LGTM!

File: recipe/meta.yaml (hunk 2 above)
Comment (Arishamays1) [reply recipe/meta.yaml]: Got it! Will do!

File: recipe/meta.yaml (hunk 1 above)
Comment (Arishamays1) [reply recipe/meta.yaml]: removing!

**Pull request merged**